joschahenningsen/TUM-Live
# Dependency updates

## Steps for testing:
- check if stuff works on the testserver, especially the calendar (major update) should be all good tho.
